library/system/io.lean

```diff
@@ -3,17 +3,17 @@ Copyright (c) 2016 Microsoft Corporation. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Luke Nelson, Jared Roesch and Leonardo de Moura
 -/
-constant {u} io : Type u → Type u
-constant io.functor : functor io
-constant io.monad   : monad io
+meta constant {u} io : Type u → Type u
+meta constant io.functor : functor io
```

Comment:
The reason I put them into meta is so that they we skip them during export, and they don't show up as additional axioms.

Personally, when I think of the meta keyword, I don't jump to "meta-programming", but rather think of the relaxed type checking requirements it entails in Lean.  You're right in that the meaning is closer to `untrusted`, and that `meta` might potentially confuse someone.

Nevertheless, I think we should cleanly separate the logically relevant and trusted parts of lean from the potentially unsound parts which are only meant to be executed.  I don't care whether we call them trusted/untrusted, regular/meta, safe/unsafe, total/partial, or whatever.  (The current meta name is the catchiest, I think, and also the one I've gotten used to.)  Whichever way we decide, it would be a good idea to have a single nomenclature everywhere (right now we use untrusted in C++, and meta in Lean code).